packages/block-editor/src/components/page-template-picker/picker.native.js

```diff
@@ -8,7 +8,7 @@ import { useDispatch, useSelect } from '@wordpress/data';
  * External dependencies
  */
 import { logUserEvent, userEvents } from 'react-native-gutenberg-bridge';
-import { Animated } from 'react-native';
+import { Animated, Dimensions, Keyboard } from 'react-native';
```

Comment:
**Note**: Didn't use [useWindowDimensions](https://reactnative.dev/docs/usewindowdimensions#__docusaurus) because there's a [bug](https://github.com/facebook/react-native/issues/26733) that will be fixed for `0.62`